src/index.js

```diff
@@ -0,0 +1,48 @@
+import './index.css';
+import { postData, getData } from './modules/getapi.js';
+
+const form = document.getElementById('add-form');
+const container = document.getElementById('scores-container');
+const name = document.getElementById('name');
+const score = document.getElementById('score');
+const refreshBtn = document.getElementById('refresh-btn');
+
+const displayScore = (person, personScore) => {
+  const tableRow = document.createElement('tr');
+  tableRow.classList.add('person-1');
+  tableRow.innerHTML = `
+   <td class="title">${person}</td>
+   <span> : </span>
+   <td class="scores">${personScore}</td>`.trim();
+
+  container.appendChild(tableRow);
+};
+
+form.addEventListener('submit', async (e) => {
+  e.preventDefault();
+  const person = name.value;
+  const personScore = score.value;
+  await postData({
+    user: person,
+    score: Number(personScore),
+  });
+
+  const scores = await getData();
+  scores.result.forEach((score) => {
+    displayScore(score.user, score.score);
+  });
+});
+
+document.addEventListener('DOMContentLoaded', async () => {
+  const scores = await getData();
+  scores.result.forEach((score) => {
+    displayScore(score.user, score.score);
+  });
+});
+
+refreshBtn.addEventListener('click', async () => {
+  const scores = await getData();
+  scores.result.forEach((score) => {
+    displayScore(score.user, score.score);
+  });
+});
```

Comment:
- [ ] I noticed that, for every click on the refresh button, the data is fetched and duplicated as many times as I click on the refresh button. Kindly fix this so the refresh button is clicked once and the data is rendered and does not have to duplicate again upon subsequent clicks. You could take a look at your render implementation again to see how you could fix that 👍